bookinfo-example/README.md

```diff
@@ -277,3 +281,5 @@ containers:
 Where I can find the authservice images?
 
 We use Github packages to host [authservice images](https://github.com/istio-ecosystem/authservice/pkgs/container/authservice%2Fauthservice).
+
+You can specify any docker image by changing the [following value](https://github.com/istio-ecosystem/authservice/blob/a7d38e4c11d79d85ce9a263c10dc2eca88a94891/bookinfo-example/authservice/values.yaml#L2).
```

Comment:
s/following value/`--set authservice.image/?

this makes sentence more self contained. no need to click to figure out the helm options.

Reply:
Changed. Is it enough for readers?